python/paddle/trainer_config_helpers/layers.py

```diff
@@ -2668,7 +2668,7 @@ def __add_evaluator__(e):
     return LayerOutput(name, LayerType.COST, parents=[input, label])
 
 def conv_operator(input, filter_size, num_filters,
-                  num_channel=None, stride=1, padding=0,
+                  num_channel=None, stride=1, padding=0, groups=1,
```

Comment:
suggest to split input into two parameters: image and kernel to make the interface clearer